src/app/starredentries/starredentries.component.html

```diff
@@ -16,110 +16,248 @@
+                          <mat-icon class="verified-icon">done</mat-icon>
+                          <span>Verified</span>
+                        </a>
+                      </mat-chip>
+                    </span>
+                  </div>
+                  <div><button mat-raised-button color="accent" class="view-button">View</button></div>
+                </div>
+              </div>
+            </a.no-underline>
+          </mat-card>
+        </div>
       </div>
     </div>
   </mat-tab>
 
-  <mat-tab label="Organizations">
+  <!-- Services not done -->
+  <mat-tab>
+    <ng-template mat-tab-label>
+      <div fxLayoutGap="0.5rem">
+        <img class="site-icons-tab" src="../../../assets/svg/sub-nav/services.svg" alt="service logo" />
+        <span>Services</span>
+        <span class="tab-display"></span>
+      </div>
+    </ng-template>
+  </mat-tab>
+
+  <mat-tab>
     <div *ngIf="starredOrganizations">
-      <div class="container">
+      <ng-template mat-tab-label>
+        <div fxLayoutGap="0.5rem">
+          <img src="../../../assets/svg/sub-nav/organization.svg" alt="organization logo" />
+          <span>Organizations</span>
+          <span class="tab-display">{{ starredOrganizations.length }}</span>
+        </div>
+      </ng-template>
+    </div>
+    <div *ngIf="starredOrganizations">
+      <div class="card-spacing">
         <div class="row">
-          <h3 class="starred-header">Starred Organizations</h3>
           <mat-card class="alert alert-info starred-header" role="alert" *ngIf="starredOrganizations.length === 0">
             <mat-icon>warning</mat-icon>
             You have no starred organizations.
           </mat-card>
         </div>
-
-        <mat-list>
-          <mat-list-item *ngFor="let organization of starredOrganizations; let last = last">
-            <h3 class="spacing" mat-line>
-              <a routerLink="/organizations/{{ organization.name }}" [matTooltip]="organization?.name" target="_blank" rel="noopener">{{
-                organization.displayName
-              }}</a>
-            </h3>
-            <span mat-line>
-              <small class="spacing" *ngIf="organization?.email"
-                >Email: <a [href]="'mailto:' + organization?.email">{{ organization.email }}</a></small
-              >
-              <small class="spacing" *ngIf="organization?.link"
-                >Website: <a [href]="organization?.link">{{ organization?.link }}</a>
-              </small>
-            </span>
-            <app-organization-starring
-              class="pull-right"
-              style="width: 100px"
-              [organization]="organization"
-              (stargazersChange)="organizationStarGazersChange()"
-            ></app-organization-starring>
-            <mat-divider></mat-divider>
-          </mat-list-item>
-        </mat-list>
+        <div *ngIf="starredOrganizations.length > 0; else noOrganizationsFound">
```

Comment:
`noOrganizationsFound` does not exist

Reply:
More generally, is it possible to use organizations component here? Or a subset of it? It probably won't work out of the box, but if we decide to change the number of orgs per row to 2 or 4, or redo the layout entirely, we'll now have to change it in two places.

I realize this won't be trivial -- the organizations component probably assumes it's displaying all orgs; you may need to extract a new component from the organizations component. But there's a lot of duplication right now.

Reply:
I think this was also mentioned in the old PR as well, but I just decided to remove the inline styles for now since I'm not sure if the starred organizations tab would look different than the organizations tab in the future.